node/node.go

```diff
@@ -658,6 +658,9 @@ func (n *Node) ConfigureRPC() {
 	rpccore.SetConsensusReactor(n.consensusReactor)
 	rpccore.SetEventBus(n.eventBus)
 	rpccore.SetLogger(n.Logger.With("module", "rpc"))
+	rpccore.MaxSubscriptionClients = n.config.RPC.MaxSubscriptionClients
```

Comment:
Should we just set a `rpccore.Config` object with `n.config.RPC` instead?

Reply:
Won't we have a DATA RACE then?

Reply:
Not if we copy